flict/__main__.py

```diff
@@ -14,21 +14,44 @@
 from flict.flictlib import flict_config
 from flict.impl import FlictImpl
 
-import os
 import sys
 
+DESCRIPTION = """
+NAME
+  flict (FOSS License Compatibility Tool)
 
-SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+DESCRIPTION
+  flict is a Free and Open Source Software tool to verify compatibility between licenses
 
-PROGRAM_NAME = "flict (FOSS License Compatibility Tool)"
-PROGRAM_DESCRIPTION = "flict is a Free and Open Source Software tool to verify compatibility between licenses"
-PROGRAM_URL = "https://github.com/vinland-technology/flict"
-BUG_URL = "https://github.com/vinland-technology/flict/issues"
-PROGRAM_COPYRIGHT = "(c) 2021 Henrik Sandklef<hesa@sandklef.com>"
-PROGRAM_LICENSE = "GPL-3.0-or-later"
-PROGRAM_AUTHOR = "Henrik Sandklef"
-PROGRAM_ATTRIBUTION = "flict is using the license compatibility matrix from osadl.org,\n  which can be found at https://www.osadl.org/fileadmin/checklists/matrix.html"
-PROGRAM_SEE_ALSO = ""
+"""
+
+EPILOG = f"""
+Return codes:
+{ReturnCodes.get_help()}
+CONFIGURATION
+  All config files can be found in
+  {flict_config.VAR_DIR}
+
+AUTHOR
+  Henrik Sandklef
+
+PROJECT SITE
+  https://github.com/vinland-technology/flict
+
+REPORTING BUGS
+  File a ticket at https://github.com/vinland-technology/flict/issues
+
+COPYRIGHT
+  Copyright (c) 2021 Henrik Sandklef<hesa@sandklef.com>.
+  License GPL-3.0-or-larer
+
+ATTRIBUTION
+  flict is using the license compatibility matrix from osadl.org,
+  which can be found at https://www.osadl.org/fileadmin/checklists/matrix.html
```

Comment:
That's not accessible anymore so we should remove this link - and we are using the library, so there isn't a need to mention it here anyway

Reply:
Fixed